liquidity/liquidity.go

```diff
@@ -404,19 +411,22 @@ func (m *Manager) saveParams(ctx context.Context, req proto.Message) error {
 func (m *Manager) loadParams(ctx context.Context) (
 	*clientrpc.LiquidityParameters, error) {
 
-	paramsBytes, err := m.cfg.FetchLiquidityParams(ctx)
+	params, err := m.cfg.FetchLiquidityParams(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("failed to read params: %v", err)
 	}
 
 	// Return early if there's nothing saved.
-	if paramsBytes == nil {
+	if params == nil {
 		return nil, nil
 	}
 
 	// Unmarshal the params.
 	req := &clientrpc.LiquidityParameters{}
-	err = proto.Unmarshal(paramsBytes, req)
+	if len(params) != 1 {
```

Comment:
nit: can move this above to the other param check.

Reply:
I think emptiness check should still come first (just as it is now).